src/Microsoft.PowerShell.Commands.Utility/commands/utility/ImplicitRemotingCommands.cs

```diff
@@ -2800,15 +2800,16 @@ private void GenerateHelperFunctions(TextWriter writer)
 
```

Comment:
@vexx32, your last commit had 5 failures in `PowerShell-CI-macos`
Restart-Computer.Should restart the local computer
```powershell
Object reference not set to an instance of an object.
at <ScriptBlock>, /Users/runner/runners/2.166.3/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Restart-Computer.Tests.ps1: line 28
28:             Restart-Computer -ErrorAction Stop | Should -BeNullOrEmpty

```
Restart-Computer.Restart-Computer Error Conditions.Should return the proper error when it occurs
```powershell
Object reference not set to an instance of an object.
at <ScriptBlock>, /Users/runner/runners/2.166.3/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Restart-Computer.Tests.ps1: line 75
75:                 Restart-Computer -ErrorVariable RestartError 2> $null

```
Stop-Computer.Should stop the local computer
```powershell
Object reference not set to an instance of an object.
at <ScriptBlock>, /Users/runner/runners/2.166.3/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Stop-Computer.Tests.ps1: line 29
29:             Stop-Computer -ErrorAction Stop | Should -BeNullOrEmpty

```
Stop-Computer.Stop-Computer Error Conditions.Should return the proper error when it occurs
```powershell
Object reference not set to an instance of an object.
at <ScriptBlock>, /Users/runner/runners/2.166.3/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Stop-Computer.Tests.ps1: line 48
48:                 Stop-Computer -ErrorVariable StopError 2> $null

```
Test-Connection.TraceRoute.TraceRoute works
```powershell
Expected exactly '10.79.10.8', but got 127.0.0.1.
at <ScriptBlock>, /Users/runner/runners/2.166.3/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Test-Connection.Tests.ps1: line 258
258:             $result[0].TargetAddress | Should -BeExactly $realAddress
```